optuna/visualization/_parallel_coordinate.py

```diff
@@ -39,8 +40,8 @@ class _DimensionInfo(NamedTuple):
     range: Tuple[float, float]
     is_log: bool
     is_cat: bool
-    tickvals: List[int]
-    ticktext: List[str]
+    tickvals: List[Union[int, float]]
+    ticktext: List[Union[int, str]]
```

Comment:
Do you think if it is possible to restrict the type since the name of the argument contains `text`?
```suggestion
    ticktext: List[str]
```

Reply:
Thanks for the comment. I fix it. PTAL.

Reply:
Thank you for your update!